server/core/swagger/v4.yaml

```diff
@@ -1417,6 +1424,44 @@ paths:
           description: 内部错误
           schema:
             type: string
+    post:
```

Comment:
Any specific reasons for making it "post" method, as far as I see it's getting the discovery information and not updating/creating anything in backend, so in my opinion it can be a "get" method

Reply:
because the request body will be large than the limitation of the query part of url